src/auth/oidc-broker.ts

```diff
@@ -108,17 +108,6 @@ export const oidcSignIn = async (args: OidcSignInArgs): Promise<OidcUser | null>
 export const revokeTokens = async (): Promise<void> => {
   // send back auth instance, so we can use it for remove and clear stale state
   const auth: AuthContextProps = getAuthInstance();
-  if (auth.settings.metadata?.revocation_endpoint) {
```

Comment:
https://github.com/DataBiosphere/terra-ui/pull/4135/files#diff-e801232acea1d7f9dd68d112a2c6ed2da031ec77d195accc951a51cb98d05202 in this PR, we removed the `revocation_endpoint` from `getConfig`, so this is code that we missed cleaning up from then. Since it is not defined, this if will never evaluate to true.